src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -1,4 +1,9 @@
-{
+var params = {
```

Comment:
I would prefer to not have the DynamoDB headers and footers included, for multiple reasons.

- We haven't decided how we are going to load the data into DynamoDB for dev, test and prod yet.
- I think this will work locally, but not for a remote DynamoDB instance.
- It's easy enough to add these to a local script to run.
- We may not always be using DynamoDB.
- It feels cleaner.